src/signals/incident/containers/IncidentContainer/debug/initialState.js

```diff
@@ -20,10 +20,10 @@ export default {
       ],
     },
   },
-  category: 'overlast-bedrijven-en-horeca',
-  subcategory: 'stankoverlast',
+  category: 'wonen',
```

Comment:
is this file used anyware? I think  or can it be removed.

Reply:
Was thinking about this, but it was already there and not create by me so :man_shrugging:

Reply:
@janjaap could you shine a light on this? Thanks!

Reply:
This file is there, I believe, to help trigger a particular category and subcategory's set of questions. I'd say remove this file.

Reply:
Thanks!